.travis.yml

```diff
@@ -5,8 +5,9 @@ jdk:
 
 android:
   components:
-    - android-22
-    - build-tools-22.0.1
+    - tools
```

Comment:
https://github.com/travis-ci/travis-ci/issues/5036#issuecomment-153297292

here is from where I took this :)